src/main/java/net/id/incubus_core/recipe/IngredientStack.java

```diff
@@ -43,6 +44,22 @@ public static IngredientStack of(Ingredient ingredient) {
         return of(ingredient, Matchbook.empty(), null, 1);
     }
 
+    public static IngredientStack ofItems(ItemConvertible... items) {
+        return of(Ingredient.ofItems(items), Matchbook.empty(), null, 1);
+    }
+
+    public static IngredientStack ofItems(int count, ItemConvertible... items) {
+        return of(Ingredient.ofItems(items), Matchbook.empty(), null, count);
+    }
+
+    public static IngredientStack ofStacks(ItemStack... stacks) {
+        return of(Ingredient.ofStacks(stacks), Matchbook.empty(), null, 1);
+    }
+
+    public static IngredientStack ofStacks(int count, ItemStack... stacks) {
+        return of(Ingredient.ofStacks(stacks), Matchbook.empty(), null, count);
+    }
+
```

Comment:
Actually wait no fuck you why are you putting count before the relevant object. This is inconsistent with ItemStack and also does NOT make me want to hit a boogie so hard my cock flies off.

Reply:
Have you ever coded in java? 🤔

Reply:
![image](https://user-images.githubusercontent.com/20349662/202157926-89321d1b-5f98-46d0-8e2a-3cfa6bab34aa.png)
L O O K A T I T

Reply:
There are few things that can make me genuinely mald so briefly in code and one of them is shit argument ordering. When I am reading through a cesspool of code shat out at 3 AM while drunk and sleep-deprived the first thing I want to know when I see some kinda stack is what the stack is for, not number 7, student gets nakidd and watches zorn.

Reply:
Not that I didn't try. I know It's ugly, but blame Java 😎

Reply:
In cdecl, variadic arguments are placed onto the stack as any other arguments would be. The length is not passed, since it can be assumed that any arguments beyond those that are required are variadic. In sysv amd64, they are also placed into registers like normal arguments, with the length passed with them. Unlike in java, where they are simply syntactic sugar for an array, they are truly first-class arguments passed to the function, with the simplicity and improved performance brought by it.

I don't know why java chose to inherit the restrictions from c. They might have been considering similarly passing them on the stack instead of creating an array, or it might just be for parity with c as with `int foo[]`. Either way, you can blame c for that more than you can java.

Reply:
I feel we have been flexed upon

Reply:
c's nuts

Reply:
I think the ordering is fine